src/components/timeline/TrackIPC.js

```diff
@@ -5,15 +5,15 @@
 // @flow
 
 import * as React from 'react';
-import explicitConnect from '../../utils/connect';
-import { getCommittedRange } from '../../selectors/profile';
-import { TimelineMarkersIPC } from './Markers';
-import { updatePreviewSelection } from '../../actions/profile-view';
-import { TRACK_IPC_MARKERS_HEIGHT } from '../../app-logic/constants';
+import explicitConnect from 'firefox-profiler/utils/connect';
+import { getCommittedRange } from 'firefox-profiler/selectors/profile';
+import { TimelineMarkersIPC } from 'firefox-profiler/components/timeline/Markers';
```

Comment:
And this import should stay like `import { TimelineMarkersIPC } from './Markers';`